src/managers/PlayerManager.js

```diff
@@ -32,10 +32,12 @@ const init = async (element) => {
   if (!videoElement) {
     videoElement = document.createElement('video')
     videoElement.setAttribute('controls', true)
-
-    const vStyles = { position: 'absolute', top: 0, left: 0, zIndex: 0 }
-    Object.assign(videoElement.style, vStyles)
     
+    const cssString = 'position: absolute; top: 0; left: 0; zIndex: 0'
+
+    // Assign the CSS string to the cssText property
+    videoElement.style.cssText = cssString
```

Comment:
Let's assign the string straight away, instead of capturing it in a const first. Saves some memory allocation